[DOC-1726] Update SAML SSO with Azure required permissions (#2977)
harness/developer-hub: docs/platform/3_Authentication/3-single-sign-on-saml.md

![](./static/single-sign-on-saml-79.png)

### Users in over 150 groups

> When users have large numbers of group memberships, the number of groups listed in the token can grow the token size. Azure Active Directory limits the number of groups it will emit in a token to 150 for SAML assertions.

Reviewer: ```suggestion
When a user has many group memberships, the number of groups listed in the token can grow the token size. Azure Active Directory limits the number of groups it will emit in a token to 150 for SAML assertions.
```

If a user is a member of a larger number of groups, the groups are omitted and a link to the Graph endpoint to obtain group information is included instead.

> To invoke the API, Harness will need **Client ID** and **Client Secret** for your registered app.

Reviewer: ```suggestion
To invoke the API, Harness requires **Client ID** and **Client Secret** for your registered app.
```

To get this information, do the following:

1. In your Azure account, go to **App registrations**.
2. Click on your app. Copy the Application (client) ID and paste it in **Client ID** in your Harness account.
3. In your Azure account, go to **App registrations**. Click **Certificates and Secrets**.
4. Click New Client Secret.

> 5. Add a description and click Add.

Reviewer: ```suggestion
5. Add a description and click **Add**.
```
